lib/nostrum/util.ex

```diff
@@ -240,6 +242,38 @@ defmodule Nostrum.Util do
      )}
   end
 
+  @doc """
+  Gets the latency of the shard connection from a `Nostrum.Struct.WSState.t()` struct.
+
+  Returns the latency in milliseconds as an integer, returning 0 if unknown.
+  """
+  @spec get_shard_latency(WSState.t()) :: non_neg_integer
```

Comment:
Do you think we should instead return something like `nil`? I think if we return 0 it could be misconstrued as there being actually 0 latency.